js/tji-datasets-explore.js

```diff
@@ -393,7 +397,8 @@ TJIChartView.prototype.filter_data = function() {
 }
 
 TJIChartView.prototype.update_charts = function(data) {
-  this.state.$count.text(data.length + ' records');
+  var count_text = ' records ' + (data.length < this.state.data.length ? '[filtered]' : 'total');
+  this.state.$count.text(data.length + count_text);
```

Comment:
nice, we could also just render exactly which filters at the top for better user affordance since we have those stored to state.